app/src/main/java/io/intrepid/contest/screens/contestcreation/namecontest/NameContestPresenter.java

```diff
@@ -0,0 +1,28 @@
+package io.intrepid.contest.screens.contestcreation.namecontest;
+
+import android.support.annotation.NonNull;
+
+import io.intrepid.contest.base.BasePresenter;
+import io.intrepid.contest.base.PresenterConfiguration;
+import io.intrepid.contest.models.Contest;
+import io.intrepid.contest.models.ContestProvider;
+
+class NameContestPresenter extends BasePresenter<NameContestContract.View> implements NameContestContract.Presenter {
+
+    NameContestPresenter(@NonNull NameContestContract.View view,
+                         @NonNull PresenterConfiguration configuration) {
+        super(view, configuration);
+    }
+
+    @Override
+    public void onViewCreated() {
+        super.onViewCreated();
+    }
+
+    @Override
+    public void updateContestName(String contestName) {
```

Comment:
That's true. I feel like this is semantics bc either way you look at it - if in one class, you call a method on an object, you're essentially telling that object what to do. I always find it bothersome because giving the view method and presenter method the same name sometimes makes it harder for me to distinguish the order in which things happened. Was the name already updated or is it time to update it. So, I'm finding I'm fond of reactive methods having the passive names (onButtonPress()) but I can definitely see the value of giving the methods the same name, that way the presenter's method definition isn't married to some specific implementation. @jmccormack200 if you have any insight on this, please share.